test/integration/consul-container/libs/consul-cluster/consul-cluster.go

```diff
@@ -0,0 +1,105 @@
+package consul_cluster
```

Comment:
Effective Go [recommends](https://go.dev/doc/effective_go#package-names) that package names are kept short and to the point:

>  package name should be good: short, concise, evocative. By convention, packages are given lower case, single-word names; there should be no need for underscores or mixedCaps.

What do you think of naming this `cluster` and the other package `node`?